Fix numeric precision issue in `testHistogram`

Failed list assertion due to numeric precision issue in new `testHistogram` test. We need to replace this with np.allclose

```
=========================== short test summary info ============================
FAILED tests/numeric_test.py::NumericTest::testHistogram - AssertionError: Lists differ: [11.0[72 chars], 22.7, 23.6, 24.5, 25.4, 26.3, 27.2, 28.1, 29.0] != [11.0[72 chars], 22.700000000000003, 23.6, 24.5, 25.4, 26.3, 27.2, 28.1, 29.0]

First differing element 13:
22.7
22.700000000000003

  [11.0,
   11.9,
   12.8,
   13.7,
   14.6,
   15.5,
   16.4,
   17.3,
   18.2,
   19.1,
   20.0,
   20.9,
   21.8,
-  22.7,
+  22.700000000000003,
   23.6,
   24.5,
   25.4,
   26.3,
   27.2,
   28.1,
   29.0]
====== 1 failed, 533 passed, 1 skipped, 7 warnings in 1219.67s (0:20:19) =======
make: *** [Makefile:504: test-python] Error 1
[Error running make test-python]
```

--- a/tests/numeric_test.py
+++ b/tests/numeric_test.py
@@ -135,8 +135,8 @@ class NumericTest(ArkoudaTest):
         self.assertEqual(20, len(ak_bins) - 1)
         self.assertEqual(20, len(ak_result))
         self.assertEqual(int, ak_result.dtype)
-        self.assertListEqual(ak_result.to_list(), np_result.tolist())
-        self.assertListEqual(ak_bins.to_list(), np_bins.tolist())
+        self.assertTrue(np.allclose(ak_result.to_list(), np_result.tolist()))
+        self.assertTrue(np.allclose(ak_bins.to_list(), np_bins.tolist()))
 
         with self.assertRaises(TypeError):
             ak.histogram([range(0, 10)], bins=1)
@@ -153,15 +153,15 @@ class NumericTest(ArkoudaTest):
         np_x, np_y = ak_x.to_ndarray(), ak_y.to_ndarray()
         np_hist, np_x_edges, np_y_edges = np.histogram2d(np_x, np_y)
         ak_hist, ak_x_edges, ak_y_edges = ak.histogram2d(ak_x, ak_y)
-        self.assertListEqual(np_hist.tolist(), ak_hist.to_list())
-        self.assertListEqual(np_x_edges.tolist(), ak_x_edges.to_list())
-        self.assertListEqual(np_y_edges.tolist(), ak_y_edges.to_list())
+        self.assertTrue(np.allclose(np_hist.tolist(), ak_hist.to_list()))
+        self.assertTrue(np.allclose(np_x_edges.tolist(), ak_x_edges.to_list()))
+        self.assertTrue(np.allclose(np_y_edges.tolist(), ak_y_edges.to_list()))
 
         np_hist, np_x_edges, np_y_edges = np.histogram2d(np_x, np_y, bins=(10, 20))
         ak_hist, ak_x_edges, ak_y_edges = ak.histogram2d(ak_x, ak_y, bins=(10, 20))
-        self.assertListEqual(np_hist.tolist(), ak_hist.to_list())
-        self.assertListEqual(np_x_edges.tolist(), ak_x_edges.to_list())
-        self.assertListEqual(np_y_edges.tolist(), ak_y_edges.to_list())
+        self.assertTrue(np.allclose(np_hist.tolist(), ak_hist.to_list()))
+        self.assertTrue(np.allclose(np_x_edges.tolist(), ak_x_edges.to_list()))
+        self.assertTrue(np.allclose(np_y_edges.tolist(), ak_y_edges.to_list()))
 
     def testLog(self):
         na = np.linspace(1, 10, 10)

